assets/js/frontend/course-video/youtube-extension.js

```diff
@@ -0,0 +1,79 @@
+( () => {
+	function handleVisibilityChange( player ) {
```

Comment:
Not critical, but I'd recommend using arrow functions in the functions just to follow the standard we're using in most parts of the project. :)

Reply:
Fixed: https://github.com/Automattic/sensei/pull/4546/commits/18d7b4e8d4884972749e2b1acbee71c2a471d70b